doc/Writerside/topics/new_outcome_flow.md

```
@@ -0,0 +1,86 @@
+
+![addType.png](addType.png)
+
+2. Добавьте предопределенный элемент справочника "инт_ПотокиДанных". {id="predefined"}
+   - Укажите имя для обращения из кода конфигурации
+   - Укажите в поле "код" [](dataflow.md#flow_id)
+   - Укажите в поле наименование представление потока для пользователя.
+3. Определите триггеры при которых должно формироваться сообщение(Прим: При каждой записи документа.)
+
+4. В обработчики событий, которые вызываются при наступлении ключевых условий добавьте регистрацию сообщения.
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)